apps/core/templates/base.html

```diff
@@ -30,6 +30,25 @@
             <link rel="alternate" hreflang="en" href="{{ page.full_url|hreflang_url:page.locale.language_code }}" >
         {% endif %}
 
+        {% comment %}
+            Twitter summary card - see https://dev.twitter.com/cards/types/summary
+            and https://dev.twitter.com/cards/getting-started
+        {% endcomment %}
+        <meta name="twitter:card" content="summary" />
+        <meta name="twitter:site" content="@wagtailcms" />
+        <meta name="twitter:title" content="{{ page.title }}" />
+        <meta name="twitter:description" content="{% if page.search_description %}{{ page.search_description }}{% else %}{{ page.listing_summary }}{% endif %}">
+        <meta name="twitter:image" content="{{ current_site.root_url }}{% static 'images/social-image.png' %}">
+
+        {# facebook opengraph tags #}
+        <meta property="fb:app_id" content="{{ settings.utils.SocialMediaSettings.facebook_app_id }}" />
+        <meta property="og:type" content="website" />
+        <meta property="og:url" content="{{ site.root_url }}{{ page.url }}" />
+        <meta property="og:title" content="{{ page.title }}" />
+         <meta name="og:image" content="{{ current_site.root_url }}{% static 'images/social-image.png' %}">
```

Comment:
```suggestion
        <meta name="og:image" content="{{ current_site.root_url }}{% static 'images/social-image.png' %}">
```

Just an extra space here I think